cmd/bootstrap/transit/cmd/push_root_block_vote.go

```diff
@@ -0,0 +1,138 @@
+package cmd
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"path/filepath"
+	"time"
+
+	"github.com/spf13/cobra"
+
+	"github.com/onflow/flow-go/cmd"
+	"github.com/onflow/flow-go/cmd/bootstrap/gcs"
+	"github.com/onflow/flow-go/consensus/hotstuff/model"
+	"github.com/onflow/flow-go/consensus/hotstuff/verification"
+	"github.com/onflow/flow-go/model/bootstrap"
+	"github.com/onflow/flow-go/model/dkg"
+	"github.com/onflow/flow-go/model/encodable"
+	"github.com/onflow/flow-go/model/encoding"
+	"github.com/onflow/flow-go/model/flow"
+	"github.com/onflow/flow-go/module/local"
+	"github.com/onflow/flow-go/module/signature"
+	"github.com/onflow/flow-go/utils/io"
+)
+
+var pushVoteCmd = &cobra.Command{
+	Use:   "push-root-block-vote",
+	Short: "Generate and push root block vote",
+	Run:   pushVote,
+}
+
+func init() {
+	rootCmd.AddCommand(pushVoteCmd)
+	addPushVoteCmdFlags()
+}
+
+func addPushVoteCmdFlags() {
+	pullCmd.Flags().StringVarP(&flagToken, "token", "t", "", "token provided by the Flow team to access the Transit server")
+	_ = pullCmd.MarkFlagRequired("token")
+}
+
+func pushVote(c *cobra.Command, args []string) {
+	log.Info().Msg("generating root block vote")
+
+	nodeIDString, err := readNodeID()
+	if err != nil {
+		log.Fatal().Err(err).Msg("could not read node ID")
+	}
+
+	nodeID, err := flow.HexStringToIdentifier(nodeIDString)
+	if err != nil {
+		log.Fatal().Err(err).Msg("could not parse node ID")
+	}
+
+	nodeInfo, err := cmd.LoadPrivateNodeInfo(flagBootDir, nodeID)
+	if err != nil {
+		log.Fatal().Err(err).Msg("could not load private node info")
+	}
+
+	// load DKG private key
+	path := fmt.Sprintf(bootstrap.PathRandomBeaconPriv, nodeID)
+	data, err := io.ReadFile(filepath.Join(flagBootDir, path))
+	if err != nil {
+		log.Fatal().Err(err).Msg("could not read DKG private key file")
+	}
+
+	var priv dkg.DKGParticipantPriv
+	err = json.Unmarshal(data, &priv)
+	if err != nil {
+		log.Fatal().Err(err).Msg("could not unmarshal DKG private key data")
+	}
+
+	randomBeaconPrivKey := priv.RandomBeaconPrivKey.PrivateKey
+	stakingPrivKey := nodeInfo.StakingPrivKey.PrivateKey
+	identity := &flow.Identity{
+		NodeID:        nodeID,
+		Address:       nodeInfo.Address,
+		Role:          nodeInfo.Role,
+		Stake:         0, // TODO: is this okay?
```

Comment:
Technically, I don't think the value for `Stake` this should matter. 
The correct value for the stake should be in the node's public node info (also available as file) but we can update this later.

Reply:
we use the value of `1000` when we stake (it is not a really used but we set stake of all nodes to 1000 when we spork).

Reply:
yeah, we might as well just use the 1000 we've been using, to ensure continuity